homeassistant/components/samsungtv/__init__.py

```diff
@@ -269,7 +269,9 @@ async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
         en_reg = er.async_get(hass)
         en_reg.async_clear_config_entry(config_entry.entry_id)
 
+        version = 2
         hass.config_entries.async_update_entry(config_entry, version=2)
```

Comment:
Wouldn't be better the following to avoid issues if one is updated and not the other ?

hass.config_entries.async_update_entry(config_entry, version=**version**)